### DISCUSSION TITLE
﻿  Error! Exception: Failed to Open PiecesOS, unable to find a proper data directory. Please restart your machine, if the problem persists please reach out to support@pieces.app, and we will get you back up and running.

### DISCUSSION THREAD

**Discussion by SatishBanchhere** [pieces-app/support#142] [Q&A]
﻿

Error!
Exception: Failed to Open PiecesOS, unable to find a proper data directory. Please restart your machine, if the problem persists please reach out to support@pieces.app, and we will get you back up and running.

**Comment by caleb-at-pieces** [comment]
Hey there @SatishBanchhere , this may do better as a github issue and / or ticket in our support system, have you filed an issue or created a support ticket yet?

**Comment by SatishBanchhere** [comment]
No I just asked this thing somewhere on gitHub then again copied it from
there and pasted it on mail.

**Reply by caleb-at-pieces** [reply]
Okay, so you have already sent this as an email to support@pieces.app?

**Comment by SatishBanchhere** [comment]
No, I don't think so.

**Reply by caleb-at-pieces** [reply]
Okay, can you please either mention this issue you are having in github issues: https://github.com/pieces-app/support/issues/new?template=bug.yml ? Or can you fill out this form to file a support ticket? https://getpieces.typeform.com/to/mCjBSIjF#page=docs-support

**Reply by SatishBanchhere** [reply]
I filled the form

**Reply by caleb-at-pieces** [reply]
Awesome, thanks for filling that out for us. I see your ticket in our support system, so one of our qualified team members will be able to help you out with this very soon!

**Reply by SatishBanchhere** [reply]
Thank you.

**Comment by SatishBanchhere** [comment]
No it is not resolved yet
